test/bpf/verifier-test.sh

```diff
@@ -6,11 +6,9 @@ set -eo pipefail
 
 DEV="cilium-probe"
 DIR=$(dirname $0)/../../bpf
-MAPTOOL=$(dirname $0)/../../tools/maptool/maptool
 # all known bpf programs (object files).
 # ALL_PROGS will be tested again source files to find non-tested bpf code
 ALL_TC_PROGS="bpf_lxc bpf_host bpf_network bpf_overlay"
-ALL_CG_PROGS="bpf_sock sockops/bpf_sockops sockops/bpf_redir"
```

Comment:
I'd defer to the @cilium/sig-datapath folks here for a final verdict, but I think we want to keep testing `bpf_sock` complexity.